contrib/README.md

```diff
@@ -37,15 +37,34 @@ It will work in the directory you called it from. So if you wish to use it, I'd
 recommend you either put it into your `$PATH` or create yourself an alias for it.
 
 Once you've set it up and have it running, you could even have a Cron job
-taken care for regular runs. Here I'd recommend the following command line:
+taken care for regular runs. With the config set up properly, the command
+for that job could look like this:
 
 ```bash
-cd /path/to/archive && mastoarch -a Me@MyInstance 2>/dev/null
+source /home/johndoe && mastoarch archive
```

Comment:
I don't understand `source /home/johndoe` – doesn't that attempt to read `/home/johndoe` as shell script?

Reply:
Oh… By replacing my login a part got lost, that should be `/home/johndoe/.profile`. Adjusted (commit coming).